packages/calcite-components/src/components/pagination/pagination.tsx

```diff
@@ -97,7 +97,7 @@ export class Pagination extends LitElement {
   @property({ reflect: true }) startItem = 1;
 
   /** Specifies the total number of items. */
-  @property({ reflect: true }) totalItems = 0;
+  @property({ reflect: true }) totalItems = 1;
```

Comment:
I don’t think this is correct since items aren’t the same as pages. Instead of changing the default, we should update the logic that handles the start item update.

Reply:
Can we revisit this in a follow-up PR? Once fixed, this PR will need to be amended and the fix PR will need to use `no changelog` label since it will be fixed within the R3 development cycle.